packages/ember-runtime/lib/system/lazy_load.js

```diff
@@ -10,6 +10,7 @@ import 'ember-runtime/system/native_array'; // make sure Ember.A is setup.
 
 var loadHooks = Ember.ENV.EMBER_LOAD_HOOKS || {};
 var loaded = {};
+export var _loaded = loaded;
```

Comment:
Is this really private? It's exported. I know JS doesn't really have 'protected' but and wondering if you can add a doc block on the export for `_loaded` What is this? private / public etc.